xcube/core/store/storepool.py

```diff
@@ -262,7 +262,27 @@ def store_instance_ids(self) -> List[str]:
     def store_configs(self) -> List[DataStoreConfig]:
         return [v.store_config for k, v in self._instances.items()]
 
-    def has_store_config(self, store_instance_id: str) -> bool:
+    def get_store_instance_id(self,
+                              store_config: DataStoreConfig,
+                              strict_check: bool = False
+                              ) \
+            -> Optional[str]:
```

Comment:
Code style:

```suggestion
    def get_store_instance_id(self,
                              store_config: DataStoreConfig,
                              strict_check: bool = False
                              ) \
            -> Optional[str]:
```
```suggestion
    def get_store_instance_id(
        self,
        store_config: DataStoreConfig,
        strict_check: bool = False
) -> Optional[str]:
```

Reply:
GitHub has a bug here: it doubles the suggestions! Only the second is valid!

Reply:
It even doubles every comment!

Reply:
I just wanted to check our code style for this issue. I did not find it, I just recall we already talked about this before.

But are you certain that the suggestion below is right? Going to the lowest indention level with the method header?

Reply:
> I did not find it, I just recall we already talked about this before.

PEP-8. And yes, we talked about this before, see https://www.python.org/dev/peps/pep-0008/#indentation

> Going to the lowest indention level with the method header?

No that's not right, it happened per accident.

Reply:
Okay, changed it according to pep8